dbms/src/Interpreters/ActionsVisitor.cpp

```diff
@@ -195,12 +195,22 @@ SetPtr makeExplicitSet(
     return set;
 }
 
-static String getUniqueName(const Block & block, const String & prefix)
+static String getUniqueName(ActionsVisitor::Data & data, const String & prefix)
```

Comment:
Maybe it's better to make it a part of Data iface `ActionsVisitor::Data::getUniqueName(const String & prefix)`